constants/common.js

```diff
@@ -102,8 +102,13 @@ export const sortAlphaAttributes = htmlString => {
 };
 
 export const isBlacklistedFileExtensions = (url, blacklistedFileExtensions) => {
-  const urlExtension = url.split('.').pop();
-  return blacklistedFileExtensions.includes(urlExtension);
+  const urlExtension = url.split('.').pop(); 
+
+  const isBlacklisted = blacklistedFileExtensions.some(extension =>
+    urlExtension.includes(extension)
+  );

```

Comment:
cmiiw `urlExtension` should only be a single value. Maybe can just
```suggestion
  const isBlacklisted = blacklistedFileExtensions.includes(urlExtension);
```

Reply:
changed thanks!